### PR TITLE
Fix #8208: OpenMetadata server supports Postgres 15

### LINKED ISSUE #8208: OpenMetadata server supports Postgres 15
Let's take a look at the init script we currently have, as it is throwing an error when trying to create the Flyway tables with a permission error on schema `public`.

### PR DESCRIPTION
**Pull request by harshach**
### Describe your changes:

Fixes #8208 

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by pmbrull — APPROVED** [review]

**Comment by harshach** [comment]
@akash-jain-10 can you check why the ES is not up and running in these tests

**Comment by akash-jain-10** [comment]
Looks like Maven CI is failing for 1 test here - 
image

**Comment by akash-jain-10** [comment]
> @akash-jain-10 can you check why the ES is not up and running in these tests

It seems this is only happening with Cypress PostgreSQL Integration Tests. I've rerun the CI again to verify if this is an intermittent issue here! The elasticsearch service seems to be running fine with other CI's in this test runs!

File: docker/postgresql/Dockerfile_postgres
[the remark this answers is not in the document]
Comment (akash-jain-10) [reply docker/postgresql/Dockerfile_postgres:1]: Fixed this issue [here](https://github.com/open-metadata/OpenMetadata/pull/12370/commits/0b988f646a0a12033037baf3cf4ba9966979900e)

File: docker/postgresql/Dockerfile_postgres [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM postgres:14
+FROM postgres:15.3
```
[the remark this answers is not in the document]
Comment (akash-jain-10) [reply docker/postgresql/Dockerfile_postgres]: Fixed this [here](https://github.com/open-metadata/OpenMetadata/pull/12370/commits/325d330d2f9f78bdd2739a1c28bff3fbad99191c)

**Review by pmbrull — APPROVED** [review]

**Pull request merged**